src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/mobileclient_stack.py

```diff
@@ -353,7 +350,9 @@ def create_custom_auth_lambda_configuration(
 
     def update_common_stack_with_test_policy(self, common_stack: CommonStack):
         stack_policy = aws_iam.PolicyStatement(
-            effect=aws_iam.Effect.ALLOW, actions=["cognito-identity:*"], resources=["*"]
+            effect=aws_iam.Effect.ALLOW,
+            actions=["cognito-identity:*", "cognito-idp:AdminCreateUser"],
```

Comment:
I know this is carry over from how things worked before, but, is there any way to reduce blast radius by only allowing specific actions on cognito-identity?

Reply:
In this specific case, it turns out this action statement is redundant: we already declare `cognito-identity:*` in another area of the stack.

More generally, it's challenging to limit `cognito-identity` because the tests themselves use many admin functions to do resource setup & tear down for the tests themselves. I've added a backlog item to reduce scope on that and other permissions: https://github.com/aws-amplify/amplify-ci-support/issues/38